Index: git/glib/configure.ac
===================================================================
--- git.orig/glib/configure.ac
+++ git/glib/configure.ac
@@ -3340,7 +3340,7 @@ AS_IF([test "x$enable_compile_warnings"
    -Wall -Wstrict-prototypes -Werror=declaration-after-statement \
    -Werror=missing-prototypes -Werror=implicit-function-declaration \
    -Werror=pointer-arith -Werror=init-self -Werror=format-security \
-   -Werror=format=2 -Werror=missing-include-dirs])
+   -Werror=missing-include-dirs])
 ])
 AC_SUBST(GLIB_WARN_CFLAGS)
 
