Index: pseudo-1.7.4/maketables
===================================================================
--- pseudo-1.7.4.orig/maketables
+++ pseudo-1.7.4/maketables
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # Copyright (c) 2008-2010, 2013 Wind River Systems, Inc.
 #
Index: pseudo-1.7.4/makewrappers
===================================================================
--- pseudo-1.7.4.orig/makewrappers
+++ pseudo-1.7.4/makewrappers
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # Copyright (c) 2008-2011,2013 Wind River Systems, Inc.
 #
