Index: elfutils-0.164/libebl/eblobjnote.c
===================================================================
--- elfutils-0.164.orig/libebl/eblobjnote.c
+++ elfutils-0.164/libebl/eblobjnote.c
@@ -42,7 +42,7 @@ void
 ebl_object_note (Ebl *ebl, const char *name, uint32_t type,
 		 uint32_t descsz, const char *desc)
 {
-  if (! ebl->object_note (name, type, descsz, desc))
+    if (! ebl->object_note (name, type, descsz, desc))
     /* The machine specific function did not know this type.  */
 
     if (strcmp ("stapsdt", name) == 0)
